app/serializers/user_serializer.rb

```diff
@@ -43,10 +23,6 @@ def max_subjects
     @model.subject_limit
   end
 
-  def uploaded_subjects_count
```

Comment:
Yeah I missed that it's now a method on user, thought the column disappeared and so it would raise if requested. Noticed it later when it popped up in the spec. LGTM and Merged.